testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -2821,6 +2822,105 @@ public void testMixedCaseTable() throws IOException {
+//                                  |  part_name text,
+//                                  |  part_number text,
+//                                  |  factory_code text,
+//                                  |  created_at timestamp,
+//                                  |  last_inspected_at timestamp,
+//                                  |  times_inspected int,
+//                                  |  est_unit_cost decimal,
+//                                  |  est_unit_cost_updated timestamp,
+//                                  |  inspection_notes text,
+//                                  |  PRIMARY KEY((hour_created, machine_code), serial_number)
+
+
+    String tableName = "widgets";
+    TableAdd tableAdd = new TableAdd();
+    tableAdd.setName(tableName);
+
+    List<ColumnDefinition> columnDefinitions = new ArrayList<>();
+
+    columnDefinitions.add(new ColumnDefinition("hour_created", "timestamp"));
+    columnDefinitions.add(new ColumnDefinition("serial_number", "text"));
+    columnDefinitions.add(new ColumnDefinition("machine_code", "text"));
+    columnDefinitions.add(new ColumnDefinition("part_name", "text"));
+    columnDefinitions.add(new ColumnDefinition("part_number", "text"));
+    columnDefinitions.add(new ColumnDefinition("created_at", "timestamp"));
+    columnDefinitions.add(new ColumnDefinition("last_inspected_at", "timestamp"));
+    columnDefinitions.add(new ColumnDefinition("times_inspected", "int"));
+    columnDefinitions.add(new ColumnDefinition("est_unit_cost", "decimal"));
+    columnDefinitions.add(new ColumnDefinition("est_unit_cost_updated", "timestamp"));
+    columnDefinitions.add(new ColumnDefinition("inspection_notes", "text"));
+    tableAdd.setColumnDefinitions(columnDefinitions);
+
+    PrimaryKey primaryKey = new PrimaryKey();
+    primaryKey.setPartitionKey(Arrays.asList("hour_created", "machine_code"));
+    primaryKey.setClusteringKey(Arrays.asList("serial_number"));
+    tableAdd.setPrimaryKey(primaryKey);
+
+    // create table
+    String body =
+            RestUtils.post(
+                    authToken,
+                    String.format("%s/v2/schemas/keyspaces/%s/tables", restUrlBase, keyspaceName),
+                    objectMapper.writeValueAsString(tableAdd),
+                    HttpStatus.SC_CREATED);
+
+    TableResponse tableResponse =
+            objectMapper.readValue(body, new TypeReference<TableResponse>() {});
+    assertThat(tableResponse.getName()).isEqualTo(tableName);
+
+    // insert a row
+    String timestamp = String.valueOf(Instant.now().getEpochSecond());
+    String serialNumber = "ABC123";
+    Map<String, String> row = new HashMap<>();
+    row.put("hour_created", timestamp);
+    row.put("serial_number", serialNumber);
+    row.put("machine_code", "DEF456");
+    row.put("part_name", "Engine");
+    row.put("part_number", "DEF456");
+    row.put("last_inspected_at", timestamp);
+    row.put("times_inspected", "2");
```

Comment:
While we are allowed to pass everything as String, we can also pass actual number. So would it make sense to try out both, either as two tests or two rows in same test?

Reply:
added an additional row insert to this test